src/components/ToolMenu/LayerTree/index.tsx

```diff
@@ -55,6 +58,59 @@ export const LayerTree: React.FC<LayerTreeProps> = ({
   } = useTranslation();
 
   const [visibleLegendsIds, setVisibleLegendsIds] = useState<string[]>([]);
+  const [layerTileLoadCounter, setLayerTileLoadCounter] = useState<any>({});
+
+  useEffect(() => {
+    if (!map) {
+      return;
+    }
+    const allLayers = MapUtil.getAllLayers(map);
+    allLayers.forEach(layer => {
+      if (layer instanceof OlLayer) {
+        const source = layer.getSource();
+        if (!source.hasListener('tileloadstart')) {
+          source.on('tileloadstart', () => {
```

Comment:
Do we need to unregister the listeners on unmount?

Reply:
good catch, added